examples/tokenreview/tokenreview.go

```diff
@@ -29,7 +29,7 @@ type authenticator struct {
 }
 
 // authenticator admits a request by the token.
-func (a *authenticator) Handle(ctx context.Context, req authentication.Request) authentication.Response {
+func (a *authenticator) Handle(ctx context.Context, req *authentication.Request) authentication.Response {
```

Comment:
What is the reason to change the signature here?

Reply:
I can revert if we want, it was to avoid copying around the Request object

Reply:
I don't have strong feelings either way, its a slight optimization that likely rarely matter. But if we do it, can we do it in a different PR so this gets a distinct release note? It seems unrelated to the rest